website/versioned_docs/version-stable/webstorm.md

```diff
@@ -18,9 +18,9 @@ For older IDE versions, please follow the instructions below.
 
 To automatically format your files using `prettier` on save, you can use a [File Watcher](https://plugins.jetbrains.com/plugin/7177-file-watchers).
 
-Go to _Preferences | Tools | File Watchers_ and click **+** to add a new watcher.
+Go to _File | Settings | Tools | File Watchers_ and click **+** to add a new watcher.
```

Comment:
Is this an actual update in the app’s menu or just a difference between macOS / Windows?

Reply:
I don't know, don't have Mac :/